front/app/components/admin/PostManager/index.tsx

```diff
@@ -452,27 +452,22 @@ const Data = adopt<DataProps, InputProps>({
       );
     }
 
+    const props = {
+      'page[size]': 10,
+      sort: 'new',
+    } as const;
+
     if (type === 'ProjectIdeas') {
       return (
-        <GetIdeas
-          type="paginated"
-          pageSize={10}
-          sort="new"
-          projectIds={projectId ? [projectId] : undefined}
-        >
+        <GetIdeas {...props} projects={projectId ? [projectId] : undefined}>
```

Comment:
Are the page size and "new" not needed here anymore?

Reply:
They are needed, and they're inside of `props` :)